app/src/main/java/com/soen490chrysalis/papilio/view/ActivitiesFragment.kt

```diff
@@ -38,7 +40,14 @@ class ActivitiesFragment : Fragment()
     ) : View?
     {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_activities, container, false)
+        val view = inflater.inflate(R.layout.fragment_activities, container, false)
+        val createActivityButton = view.findViewById<androidx.appcompat.widget.AppCompatButton>(R.id.activities_create_activity_button)
+
+        createActivityButton?.setOnClickListener {
```

Comment:
Run code formatter

Reply:
Running the formatter on this causes no changes, but when manually putting the open bracket on the next line, android studio throws an error 
![image](https://user-images.githubusercontent.com/58053432/202964699-e861a3e8-d17f-4f79-8707-de4d8d9e46c9.png)